### DISCUSSION TITLE
Contribution guidelines and ToDo

### DISCUSSION THREAD

**Discussion by scarecrow1123** [AI4Bharat/Shoonya#10] [Q&A] [answered]
Hello,
Congratulations on the release of this excellent project! 

Is there a set of Contribution Guidelines / ToDo's / Roadmap ? That'd encourage public contributions and grow a community around this project.

**Comment by aravinth** [comment, marked as answer]
Hi Anand,

Thank you for your wishes. We are working on the  Contribution Guidelines. Will keep you posted around this. 
Regarding ToDo's and Roadmap, we are currently tracking via JIRA board. We will try to expose it via a wiki page.
